docker.md

```diff
@@ -1,145 +1,253 @@
-# Running API in local docker
+# Docker Stack Goals
 
```

Comment:
I think this could use a 'quickstart' section that cuts straight to what you need to do to:
- run a local api that connects to the live database
- provision and setup a local database
- run a local api that talks to the local database you've set up

Reply:
possibly also something about deleting the docker-entrypoint-initdb folder(s) if something goes wrong?